src/components/error-boundary.jsx

```diff
@@ -8,19 +8,18 @@ class ErrorBoundary extends React.Component {
     super(props);
     this.state = {
       error: false,
-      info: null,
     };
   }
 
-  componentDidCatch(error, info) {
-    this.setState({ error, info });
+  componentDidCatch(error) {
+    this.setState({ error });
   }
 
   render() {
-    const { error, info } = this.state;
+    const { error } = this.state;
     const { children } = this.props;
     if (error) {
-      return (<Splash error={error} componentStack={info.componentStack} />);
+      return (<Splash error={error} />);
```

Comment:
The output will be actually same as before, since the error itself contains all the info we wanted to remove. I removed the componentStack anyway, since I think that a user of catwalk never wants to see this.. :)